JesusOmarR/react-apprenticeship-capstone3
# Capstone3

## Description
```
React apprenticeship third mini challenge.
I created a Main Page where the notes that are not archived will be displayed
Created a NotesComponent to display the info of each note
Created a Modal to edit each note 
Created an Archive page for display all the archived notes
Created a search input to look for specific notes
Implemented Fire base on the Login Page
```
## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
```
Run all the test suites with npm run tests and see if the test have a good implementation
the test user is : prueba@prueba.com 12345678

## What is the deliverable?
```
React apprenticeship  third minichallenge
```
## Screenshots and links
```
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/97203630/151916420-846a51cc-1b7c-4ded-b495-ab148cdab041.png">

```
## Checklist
- [ x] I added the necessary documentation, if appropriate.
- [ x] I added tests to prove that my fix is effective or my deliverable works.
- [x ] I reviewed existing Pull Requests before submitting mine.
- [ x] I have commented my code, particularly in hard-to-understand areas
